### PR TITLE
Update repo name and version in bower.json and package.json

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Currently:
```
"name": "assets"
"version": "0.0.0"
```
Suggesting:
```
"name": "origin-web-console"
"version": "1.3.0"
```
To match the repo name & the current version of origin when we broke off.

@jwforres @spadgett

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "assets",
-  "version": "0.0.0",
+  "name": "origin-web-console",
+  "version": "1.3.0",
```
Comment (jwforres) [inline bower.json]: technically we'd be 1.3.0-alpha or whatever that format is since we haven't actually cut a release of 1.3.0

Comment (benjaminapetersen) [inline bower.json]: Ah, good point.  `1.3.0-alpha` looks right by [#9 on semver](http://semver.org/#spec-item-9). I can update.

**Comment by benjaminapetersen** [comment]
`1.3.0-alpha` updated.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**